templates/assets/stylesheets/common/_variables.sass

```diff
@@ -0,0 +1,8 @@
+$screen-md: 992px !default
```

Comment:
myslel jsem si to, ale nevěděl jsem, jestli v tom není nějaký záměr :smile: asi klidně jo, je to hodně specifické návrh od návrhu...